src/components/dashboard/main-content/course-enrollments/course-cards/move-to-in-progress-modal/MoveToInProgressModal.jsx

```diff
@@ -67,26 +69,41 @@ const MoveToInProgressModal = ({
 
   return (
     <MoveToInProgressModalContext.Provider value={contextValue}>
-      <Modal
+      <ModalDialog
```

Comment:
nit: i just realized we could probably also be using `StandardModal` ([docs](https://paragon-openedx.netlify.app/components/modal/standard-modal/)) in all these `ModalDialog` examples, too, as we're not doing anything unique beyond the default behavior of `StandardModal`. The biggest benefit is that some of the modal sub-components like `ModalDialog.Header` and `ModalDialog.Title` are abstracted into `StandardModal` versus needing to render it ourselves.

I have no strong opinions if you want to keep it as `ModalDialog` though. Let me know what you think :)